buddyboss/buddyboss-platform
# The students can access the course group even after the expiration of the membership.

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #744 .

### How to test the changes in this Pull Request:

1. Go to MemberPress > Memberships and create a paid membership
2. Register a user with that membership
3. Log in as a newly registered user and check the groups, course, and membership section in the front end, you will see the user can access the course and group associated with that membership
4. Go to backend > MemberPress > Subscriptions > expire the membership of the user manually
5. you can see the user can still be viewed as a member of the course group

### Proof Screenshots or Video
http://somup.com/cYiVluQpmO



### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
